sklearn/impute.py

```diff
@@ -466,6 +448,16 @@ class IterativeImputer(BaseEstimator, TransformerMixin):
         ``return_std`` in its ``predict`` method if set to ``True``. Set to
         ``True`` if using ``IterativeImputer`` for multiple imputations.
 
+    max_iter : int, optional (default=10)
+        Maximum number of imputation rounds to perform before returning the
+        imputations computed during the final round. A round is a single
+        imputation of each feature with missing values. The stopping criterion
+        is met once `abs(max(X_i - X_{i-1}))/abs(max(X[known_vals]))` < tol.
```

Comment:
you might need to clarify that X_i is X at iteration i. Might be better to use t to index time?